packages/app-staking/src/Overview/CurrentList.tsx

```diff
@@ -20,17 +22,66 @@ type Props = I18nProps & {
   recentlyOffline: RecentlyOfflineMap
 };
 
-class CurrentList extends React.PureComponent<Props> {
+type State = {
+  filter: ValidatorFilter,
+  filterOptions: Array<{ text: React.ReactNode, value: ValidatorFilter }>
+};
+const Wrapper = styled.div`
+  .filter {
+    display: flex;
+    justify-content: flex-end;
+    margin-bottom: 0.75rem;
+
+    > div {
+      max-width: 35rem;
+    }
+  }
+`;
+
+class CurrentList extends React.PureComponent<Props, State> {
+  state: State;
+
+  constructor (props: Props) {
+    super(props);
+
+    const { t } = props;
+
+    this.state = {
+      filter: 'all',
+      filterOptions: [
+        { text: t('Show all validators and intentions'), value: 'all' },
+        { text: t('Show only my nominations'), value: 'iNominated' },
+        { text: t('Show only with nominators'), value: 'hasNominators' },
+        { text: t('Show only without nominators'), value: 'noNominators' },
+        { text: t('Show only with warnings'), value: 'hasWarnings' },
+        { text: t('Show only without warnings'), value: 'noWarnings' }
+      ]
+    };
+  }
+
   render () {
+    const { t } = this.props;
+    const { filter, filterOptions } = this.state;
     return (
-      <div className='validator--ValidatorsList ui--flex-medium'>
-        <div className='validator--current'>
-          {this.renderCurrent()}
+      <Wrapper>
+        <div className='filter'>
+          <Dropdown
+            help={t('Select which validators/intentions you want to display, all, only the ones you nominated, only the ones that has/no nominators, only the ones that has/no warnings.')}
```

Comment:
The `help` text has some grammatical errors. Once you click on the dropdown the options are pretty straight forward so you don't really need to list them all in the help text. I'd keep it short:
`Select which validators/intentions you want to display.`

Reply:
Totally agree, thanks!